lib/erl_interface/test/ei_format_SUITE_data/ei_format_test.c

```diff
@@ -190,6 +197,16 @@ TESTCASE(maps)
     send_format("#{ [a] => 1 , 2 => [b,c] , \"3\" => {c,d} }");
     send_format("#{a=>[],[]=>#{2=>d}}");
     send_format("#{ a => [] , [] => #{ 2 => d } }");
+
+    /* Incorrect map syntax */
+    if (send_format("#{a= >1}") == 0) report(0);
+    if (send_format("#{,a=>1}") == 0) report(0);
+    if (send_format("#{a=>1,}") == 0) report(0);
+    if (send_format("#{=>1}") == 0) report(0);
+    if (send_format("#{a=>1,,b=>2}") == 0) report(0);
+    if (send_format("#{a=>}") == 0) report(0);
+    if (send_format("#{a=>1=>2=>3}") == 0) report(0);
```

Comment:
if (send_format("#{a=>1,b}") == 0) report(0);

Reply:
if (send_format("#{b=}") == 0) report(0);

Reply:
if (send_format("#{a>}") == 0) report(0);

Reply:
Sure, add them all if you like.